scripts/xtensa-build-all.sh

```diff
@@ -7,8 +7,8 @@ set -e
 
 # Platforms with a toolchain available in the latest Docker image and
 # built by the -a option.
-DEFAULT_PLATFORMS=(  byt cht bdw hsw apl skl kbl cnl sue icl jsl \
-                    imx8 imx8x imx8m imx8ulp tgl tgl-h rn mt8186 mt8195 )
+DEFAULT_PLATFORMS=(  byt cht bdw hsw tgl tgl-h apl skl kbl cnl sue icl jsl \
+                    imx8 imx8x imx8m imx8ulp rn mt8186 mt8195 )
 
```

Comment:
This is unrelated to commit message description. I see you reordered some platforms? Please move this in a separate commit with proper explanation.

it doesn't look like an alphabetical reorder.

Reply:
github description will be lost after this is submitted to sof repo. Lets add every useful information inside the git commit messages.

Reply:
Agreed. I used to copy/paste/diverge commit messages into the github description. I stopped when there is more than one commit; now I'm only providing a very short summary and pointing at the commit messages.

Reply:
This change doesn't seem so important as to separate it into a separate commit. Besides, its justification would require a reference to the commit in which it is currently located.